repair/row_level.cc

```diff
@@ -450,6 +450,7 @@ class repair_writer {
     // written.
     std::vector<bool> _partition_opened;
     streaming::stream_reason _reason;
+    named_semaphore _sem{1, named_semaphore_exception_factory{"repair_writer"}};
```

Comment:
Should this be per node? This will now serialize all writes to all node-specific queues.

Reply:
It does not matter because 1) on the repair master, the apply_rows runs one node after another 2) on the repair follower, there is only at most one peer node. That's why I am using a single sem for all the peer nodes.